README.md

```diff
@@ -1,3 +1,20 @@
-# pynb
+# Requirements
+
+1. passwordless ssh access:
+
+	`$ ssh-keygen`
+	`$ ssh-copy-id -i hpc.nbi.ac.uk`
+
+   After this, `$ ssh hpc.nbi.ac.uk` shouldn't ask for password.
+
+2. conda environment with dependencies:
+
+	`$ name=something123`
+	`$ conda create -p $HOME/$name jupyter paramiko`
+	`$ source activate $HOME/$name`
+
+# Usage
+
+	`$ headNode=v0523`
```

Comment:
I wouldn't add the headnode to the documentation as it depends on the environment and isn't great practice to have nodenames floating around on the internet